### DISCUSSION TITLE
2024 3Q <성장 수집가> 황윤희의 OKR

### DISCUSSION THREAD

**Discussion by yunhee1** [Pseudo-Lab/growth-collector#9] [우리들의 성장 목표!]
O: 기술(AI,개발)과 비즈니스에 대한 이해도가 높은 PO 되기

KR1: RAG, responsible AI, Chain of Thought 공부해서 블로그에 올리기
KR2: 최소 주 1회 PM(PO)/AI 관련 유튜브 보고 블로그 작성하기
KR3: 책 '그로스 해킹' 스터디 및 블로그 작성

**Comment by yunhee1** [comment]
Action Item (2024.08.23~ 2024.09.05)

1. RAG 공부 및 블로그 리뷰
2. [기획 유튜브](https://www.youtube.com/watch?v=ZSzWRCz5hHc) 보고 블로그 리뷰
3. '그로스 해킹' 챕터 2까지 읽고 블로그 리뷰
4. 인공지능 회사의 PM 10인 조사

**Reply by yunhee1** [reply]
- RAG 공부한 것 블로그에 올리지 못했습니다.. (반성)
- 기획 유튜브.. 못봤습니다..(반성)
- 그로스 해킹 읽는 중입니다...(반성)
- PM리서치 .. 하겠습니다.(반성)

**Reply by seriousran** [reply]
2주간도 고생하셨습니다!! 프라이머 해커톤에서 좋은 결과 있으시길 바라겠습니다!!

**Reply by ZeVicTech** [reply]
프라이머 해커톤 열심히 준비한만큼 좋은 결과 나오면 좋겠습니다~!

**Comment by yunhee1** [comment]
Action Item (2024.09.06~ 2024.09.18)

1.RAG 블로그에 작성하기
2. '그로스 해킹' 챕터 3까지 읽고 블로그 리뷰
3. 인공지능 회사의 PM 5인 조사

**Reply by soncy0311** [reply]
https://acl2023-retrieval-lm.github.io/

**Reply by yunhee1** [reply]
그로스 해킹 
ch1 - https://luving-me.tistory.com/30
ch2 - https://luving-me.tistory.com/31

**Reply by seriousran** [reply]
우리 성장 수집가에도 적용할 수 있는 <가설, 실험, 평가>를 만들어보면 어떨까요?!

**Reply by yunhee1** [reply]
오 좋네요! 다음 세션 때 준비해와보겠습니다🤔

**Reply by soncy0311** [reply]
제가 RAG 처음 이해할때 도움이 많이 됐던 게시글입니다!
https://discuss.pytorch.kr/t/rag-1-2/3135
https://discuss.pytorch.kr/t/rag-2-2/3160

**Comment by yunhee1** [comment]
Action Item (2024.09.19~ 2024.10.02)

1. PM 관련 [유튜브](https://www.youtube.com/watch?v=FxlsMTTb0v4&t=31s) 보고 요약
2. '그로스 해킹' 챕터 3-2 까지 읽고 블로그 리뷰
3. 유명 AI 회사의 PM 2인 조사
4. 성장 수집가에 적용할 수 있는 가설, 실험 만들기

**Comment by yunhee1** [comment]
Action Item(2024.10.03 ~ 2024.10.17)
1. 성장 수집가 링크드인 포스팅
2. '그로스해킹' 챕터 3 모두 읽고 블로그 리뷰
3. 온보딩 프로세스 작성해가기 
4. 2024.12.12까지 목표 작성해가기

**Reply by yunhee1** [reply]
1. 링크드인 포스팅 완료 했습니다(24.10.04)

2. 책은 모두 읽었는데 포스팅을 못 했습니다.. >> 다음 회차까지 반드시 완료 예정!

3. 온보딩 프로세스 (아이디어)
- 인사, 자기소개
- 성장 수집가에 오게 된 계기
- 투썸 제스처 알려주기 - 신고식
- 기존 멤버들의 OKR 공유 및 개인 OKR 수립

4. 2024.12.12까지 개인 목표
- 그로스해킹 다 읽고 포스팅 완료
- 토스 세션 7개 다 듣고 포스팅
- ADSP 꼭 따기!

2024.10.03~2024.10.17 동안 있던 일
- 2024 당근 컨퍼런스 다녀옴 (24.10.07)
- 대학생 어플리케이션 개발팀 멘토링 (24.10.08)

**Comment by yunhee1** [comment]
Action Item(2024.10.21 ~ 2024.10.27)
1. 성장 수집가 링크드인 포스팅 올리기 - 10/23(수) 자정까지
2. 토스 세션 1개 듣고 정리 - 10/26 저녁까지
3. 그로스해킹 3-1,2 블로그 포스팅 - 10/26(토)에 시작, 10/27(일) 오전까지 완료

**Reply by yunhee1** [reply]
Action Item(2024.10.22 ~ 2024.10.30)

*지난 주에 2,3번 수행을 못 해서, 이번 주에 더해서 하겠습니다.
1. 토스 세션 1개 듣고 정리 - 10/29(화) 자정까지
2. 그로스해킹 3장까지 모두 정리

**Reply by SunandBean** [reply]
## 유튜브 영상 트랜스크립트 요약

핵심 요점:

* 아하 모멘트를 통한 리텐션 개선: 서비스의 핵심 가치를 경험하는 순간을 정의하고, 유저가 그 순간을 경험하도록 유도하여 리텐션을 높이는 전략.
* 액티베이션의 진정한 의미: 회원 가입이나 퍼널이 아닌, 고객이 겪는 첫 번째 행복한 경험(The First Happy Experience)을 의미하며, 아하 모멘트와 동일.
* 액티베이션 퍼널의 중요성: 서비스의 성장에 필수적인 요소로, 인플로우, 리텐션, 바이럴 성장 등 모든 지표에 영향을 미침.

주요 섹션:

1. 아하 모멘트 찾기

• 아하 모멘트: 서비스의 핵심 가치를 경험하는 순간, 리텐션에 큰 영향을 미침.
• 아하 모멘트 찾는 방법:
• 직관/노가다: 서비스의 특성을 고려하여 직관적으로 찾거나, 룰을 설정하여 찾아봄.
• 데이터 분석: Shap Value 등을 활용하여 리텐션에 영향을 미치는 액션 후보군을 찾아냄.
• 아하 모멘트 검증: RPV(Retain Probability Value)와 교차값을 이용하여 액션과 리텐션의 인과관계를 분석.
• RPV: 액션을 한 유저 중 리텐션된 유저의 비율, 95% 이상이면 액션이 리텐션에 큰 영향을 미침.
• 교차값: 액션을 한 유저와 리텐션된 유저의 교집합 비율, 높을수록 액션과 리텐션의 연관성이 높음.
• 아하 모멘트 정의: RPV가 95% 이상이고 교차값이 최대인 액션을 찾아, 액션 횟수와 리텐션의 관계를 분석하여 아하 모멘트를 정의.

2. 액티베이션 개선

• 액티베이션: 고객이 겪는 첫 번째 행복한 경험, 아하 모멘트와 동일.
• 액티베이션 개선의 목표: 더 많은 유저가 더 빨리 더 손쉽게 아하 모멘트를 경험하도록 만드는 것.
• 액티베이션 개선 전략:
• 퍼널 개선: 전환율을 높이는 것보다, 퍼널 통과 시간을 단축하는 것이 중요.
• 아하 모멘트 경험 촉진: 유저가 핵심 가치를 경험하도록 유도하는 전략 수립.
• 액티베이션 퍼널 분석: 퍼널 단계별 전환율을 분석하여 문제점을 파악하고 개선.
• 연간 분석: 액티베이션 퍼널 통과 유저와 미통과 유저의 차이점을 분석하여 개선 방향을 모색.

3. 액티베이션 퍼널의 중요성

• 액티베이션 퍼널은 서비스의 성장에 필수적인 요소.
• 액티베이션 퍼널 개선은 인플로우, 리텐션, 바이럴 성장 등 모든 지표에 영향을 미침.
• 액티베이션 퍼널의 통과율은 서비스가 도달할 수 있는 MAU를 결정하는 중요한 요소.
• 액티베이션 퍼널은 화면 단위가 아닌 그래프 기준으로 분석해야 문제점을 정확히 파악 가능.

4. 결론

• Product/Market Fit을 찾은 후에는 리텐션과 액티베이션 개선에 집중해야 함.
• 아하 모멘트와 액티베이션 퍼널을 정의하고 개선하는 것은 서비스 성장에 필수적인 과정.
• 댓글을 통해 활발한 의견 교환과 논의를 통해 더 좋은 방법을 찾아가는 것이 중요.

**Reply by yunhee1** [reply]
직접 만드신 요약봇으로 만들어진 요약글(대단하다..!) 감사합니다 선빈님-!

**Reply by seriousran** [reply]
이번주 윤희님이 먼저 Action Item 공유해주셔서 리마인드가 정말 잘 되었었어요! 감사합니다! 그럼에도 저는 못했지만... 다음 주에는 꼭! 실행해보겠습니다!

**Reply by soncy0311** [reply]
윤희님 덕분에 저도 한주는 액션 아이템을 쓸 수 있었네요!! 다음 2주도 화이팅입니다!

**Reply by yunhee1** [reply]
1. 링크드인 작성 > 완료
2. 토스 세션 > 영상 보는 중!
3. 그로스 해킹 > 3장 모두 읽었지만 글 작성이 어렵네요 ㅠ

이번 수행을 통해 제가 글 쓰기를 굉장히 두려워하는 사람이라는 것을 깨달았습니다.
특히 다른 분들이 보신다고 생각하니, 모난 글을 쓰고 싶지 않아서 더 잘 써지지 않았던 것 같습니다.
이번 모임에서 해주신 조언들 (일단 쓰고, ctrl+c,v ㅎㅎ)을 통해 글쓰기의 두려움을 해소해보겠습니다-!

**Comment by yunhee1** [comment]
Action Item(2024.11.04 ~ 2024.11.10)
1. SK AI summit(11/4) 참가 후 후기 작성 - 11/10(일) 자정까지
2. 성장 수집가 링크드인 올리기 - 11/10(일) 자정까지
3. 토스 세션 1,2 듣고 정리 - 11/10(일) 자정까지
4. 아비투스 구매 및 1장 읽기 - 11/9(토) 자정까지
5. 그로스해킹 3-1,2 블로그 정리 - 11/10(일) 자정까지

**Reply by yunhee1** [reply]
4. 아비투스 구매 및 1장 읽기 완료 - 12/10(일)

**Reply by yunhee1** [reply]
Action Item (24.11.11 ~ 24.11.14)
1. Grand Gathering 준비: 발표 자료 준비 - 11/13(수) 자정까지
2. 아비투스 2장 읽고 정리 - 11/11부터 읽고 13일에 정리 완료

**Reply by soncy0311** [reply]
윤희님 요번주도 화이팅입니다!

**Comment by yunhee1** [comment]
Action Item(2024.11.16 ~ 2024.11.23)
1. 가짜연구소 1st Grand Gathering - 성장수집가 부스 운영 준비

**Reply by yunhee1** [reply]
Action Item(2024.11.25 ~ 2024.12.01)
1. 가짜연구소 Grand Gathering 회고
2. 아비투스 2장 다 읽고 요약 정리
3. 그로스 해킹 4장까지 읽기

**Comment by yunhee1** [comment]
Action Item(2024.12.02 ~ 2024.12.08)
1. 아비투스 다 읽고, 정리 (노션)
2. 그로스해킹 다 읽기, 느낀점 정리 (블로그)
3. 1st Grand Gathering 회고 작성 (링크드인)

**Reply by SunandBean** [reply]
이번주도 화이팅입니다~!

**Reply by rlaalsrl0922** [reply]
화이팅 입니다 :)

**Reply by soncy0311** [reply]
그로스 해킹 다 끝내는 건가요!!! 화이팅입니다!

**Reply by yunhee1** [reply]
다들 응원해주셔서 영광이네요 🏄‍♂️ 감사합니다!

Action Item(2024.12.09 ~ 2024.12.15)
1. 토스 PO 세션 1~3 보고 정리 (블로그)
